Sources/CombustionBLE/Probe.swift

```diff
@@ -85,7 +92,7 @@ public class Probe : Device {
     
     /// Pretty-formatted device name
```

Comment:
Should we just remove this?  Since "name" logic will be inside the app rather than the framework?

Reply:
Oh, good question. I think the `.name` call has to stay for API compatibility. I hadn't thought about the fact that the editable device name would be outside the framework.

Reply:
Leaving it per discussion.